model/src/main/kotlin/utils/LicenseResolver.kt

```diff
@@ -55,25 +55,16 @@ internal class LicenseResolver(private val ortResult: OrtResult) {
 
         val matchedFindings = ortResult.getScanResultsForId(id).flatMap { scanResult ->
             val rawLicenseFindings = scanResult.summary.licenseFindings
-            val copyrightFindings = scanResult.summary.copyrightFindings
+            val copyrightFindings = scanResult.summary.copyrightFindings.filter { copyright ->
```

Comment:
Commit message: *How* does this improve performance? By doing what?

Reply:
By (1) giving a smaller input the the FindingsMatcher and by (2) eliminating the need of going through the findings again and make copies of them with the copyrights stripped. This one IMO is quite obvious from the removed code.

Reply:
If you're creating time pressure to get reviews, you have to be prepared that people only have time to skim through the code, and cannot take the time to understand each change thoroughly.

Reply:
I assume you are referring to my ask for a "timely review" in the chat. Just to clarify: I didn't mean to create time pressure. My intention was to indicate that a timely review for this PR would help me a lot.

Is there anything you suggest to do differently? Are you maybe asking to put more information into the commit message?

Reply:
To me that's a superfluous message in the chat then as I always review PRs as fast as I can. So asking for a "timely review" beyond that indeed creates time pressure on my side.

Reply:
That was quite unfortunate then as I intended to write it into the core devs chat. Sorry!

Reply:
> By (1) giving a smaller input the the FindingsMatcher and by (2) eliminating the need of going through the findings again and make copies of them with the copyrights stripped.

It's a pity that you spent the time writing this here instead of in the commit message, where everyone would benefit from the explanation.